src/pages/authorization/agent-authorization/index.mdx

```diff
@@ -9,7 +9,7 @@ desc: "Learn how to authorize your app to send a sample request using our Messag
 
 # Introduction
 
-To call LiveChat APIs, you need to authorize your calls with an access token. This document describes three authorization flows that let you acquire a token:
+The authorization methods described in this document may be used to create the LiveChat and HelpDesk products. To call either of these APIs, you need to authorize your calls with an access token. This document describes three authorization flows that let you acquire a token:
```

Comment:
```suggestion
The authorization methods described in this document may be used to work with both LiveChat and HelpDesk. To call the APIs of either of these products, you need to authorize your calls with an access token. This document describes three authorization flows that let you acquire a token:
```

Reply:
1. "create LiveChat and HelpDesk products isn't accurate"
2/ I'm not sure if _"To call the APIs of either of these products"_ is clear, though. It could probably be phrased in a better way

Reply:
`You need to authorize your calls with an access token to call the APIs of any of these two products.` - WDYT?